src/lib/kw_db.sh

```diff
@@ -174,14 +174,10 @@ function remove_from()
     return 22 # EINVAL
```

Comment:
In this commit message, you missed a colon in `src: lib: kw_db Update database handling functions ` between `kw_db` and `Update`.